BitwardenShared/UI/Auth/CompleteRegistration/CompleteRegistrationProcessorTests.swift

```diff
@@ -483,6 +483,37 @@ class CompleteRegistrationProcessorTests: BitwardenTestCase {
         XCTAssertEqual(coordinator.loadingOverlaysShown, [LoadingOverlayState(title: Localizations.creatingAccount)])
     }
 
+    /// `perform(_:)` with `.completeRegistration` navigates to login if the create account request
+    /// succeeds, but login fails.
+    @MainActor
+    func test_perform_completeRegistration_loginError() async throws {
```

Comment:
⛏️ Shall the same test be added for when `services.authRepository.unlockVaultWithPassword` throws? I believe the same behavior is expected in that case.